pupil-api/src/controllers/auth.controller.ts

```diff
@@ -1,9 +1,27 @@
 
 import { Request, Response } from 'express'
+import * as winston from 'winston'
+
+import * as apiResponse from './api-response'
+import { pupilAuthenticationService } from '../services/pupil-authentication.service'
 
 class AuthController {
   async postAuth (req: Request, res: Response) {
-    return res.status(501).send('Not yet implemented')
+    const contentType = req.get('Content-Type')
+    if (!(contentType === 'application/json' || contentType === 'application/json; charset=utf-8')) {
```

Comment:
Maybe you can use `req.is(‘application/json’)` which catches both (including other charsets, although that most likely won’t happen)?